.github/workflows/python-api-tests.yml

```diff
@@ -16,6 +16,7 @@ on:
 
 env:
   TAIGA_SECRET_KEY: secret
+  TAIGA_DEBUG: true
```

Comment:
why this?

Reply:
To guarantee that the tests in the CI pass with debug to true. This will get us better traces when there is an error, especially if any api tests fail with a 500 (according to [this code](https://github.com/taigaio/taiga/blob/main/python/apps/taiga/src/taiga/exceptions/api/middlewares.py#L22)).